letsdrill.rb

```diff
@@ -1,24 +1,31 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  case
+    when integer > 89
+      result = "A"
+    when integer > 79
+      result = "B"
+    when integer > 69
+      result = "C"
+    when integer > 59
+      result = "D"
+    when integer < 60
+      result = "F"
+  end
```

Comment:
Looks better.  But: when is the the `result` local variable being used?

Reply:
I was actually uncertain about this one, so I cheated and wrote more tests below the line it said not to write more code to check myself.

I still don't *fully* understand Ruby `case` statements, but it seems like once a case is found, it's done running and exits the case, leaving `result = __` as the final executed line.

Reply:
Right, in ruby you don't need a `break` or anything.  My point was just that you could drop all of the `result = `'s, and it would still work.

Reply:
i.e., replace `result = "F"` with `"F"`, etc.

Reply:
Ah, right, because that still returns "F", that makes sense, but it would never have occurred to me to write a string with no variable assignment. The Java still runs deep I guess

Reply:
So then, is having multiple exit points from a method considered pretty standard practice in Ruby/at CMM?

Reply:
Yeah.  You'll often see what are called 'guard clauses', such as the first line here:
```ruby
  def do_a_thing!
    return if the_thing.not_that_kind_of_thing?
    the_thing.really_do_it!
  end
```